src/app/[lng]/buy/page.tsx

```diff
@@ -1,7 +1,26 @@
 import App from '@/app/ui/app/App';
+import { WidgetContainer, Widgets } from '@/components/Widgets';
+import { OnRamper } from '@/components/OnRamper';
+import { getCookies } from '@/app/lib/getCookies';
 
-const Page = async () => {
-  return <App starterVariant="buy" />;
+const Page = () => {
+  const variant = 'buy';
```

Comment:
We have this TabsMap.Buy.variant. Wdyt?

Reply:
https://github.com/lifinance/jumper.exchange/pull/1023#discussion_r1639450829